packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -1416,6 +1424,22 @@ class _SelectableRegionContainerDelegate extends MultiSelectableSelectionContain
   }
 }
 
+/// A mixin to standardize the usage of [selectionSeparator].
+mixin MultiSelectableSeparator {
```

Comment:
should this built in to the MultiSelectableSelectionContainerDelegate instead? does not seem necessary to mix this onto widget

Reply:
I made this a mixin to share it with other classes. If we aren't exposing this property on anything else we can remove it, but we do probably need it for the Stack implementation

Reply:
I went ahead and removed this because of the Flex changes